src/components/Select/Select.tsx

```diff
@@ -293,8 +293,8 @@ export const Select: FC<SelectProps> = React.forwardRef(
                     <Tooltip
                         classNames={styles.selectTooltip}
                         content={value.text}
+                        placement={'top'}
```

Comment:
does our tooltip not support auto placement based on position within the viewport? Seems like you are having to push the storybook example away from the edge to support this, and I would image we might hit scenarios where this will be problematic.

Reply:
It doesn't. Will need to implement this. However, in this scenario the tooltip would be hidden by the dropdown anyways, even with edge detection.